SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -166,10 +166,7 @@ bool FSpatialGDKEditor::LoadPotentialAssets(TArray<TStrongObjectPtr<UObject>>& O
 			return false;
 		}
 		const FString PackagePath = Data.PackagePath.ToString();
-		if (!PackagePath.StartsWith("/Game"))
-		{
```

Comment:
@mattyoung-improbable - Can you verify this part of the change please? This will result in a slight increase in SchemaGen time.

Reply:
I'll move the removal of this block to a separate PR and make it optional.  That change will have to go in before Spatial Debugger.

Reply:
Chatted with Matt - he's ok with this going in as is.